client/components/Product.js

```diff
@@ -21,7 +21,7 @@ const Product = ({ product, currentUser }) => {
               src={product.images.image1 || product.image}
               layout="fill"
               objectFit="cover"
-              priority
+              // priority
```

Comment:
try to optimize images on the home page to be lazy-load for better website performance.